tests/unit/test_project_rest.py

```diff
@@ -258,6 +258,16 @@ def check_connection(videos, project):
             video_resolver(video_ids),
             project_resolver(project_dict['metadata']['_deposit']['id']))
 
+        # check indexed record
+        RecordIndexer().process_bulk_queue()
+        sleep(2)
+        res = client.get(url_for('invenio_records_rest.recid_list',
+                                 headers=json_headers))
+        assert res.status_code == 200
+        data = json.loads(res.data.decode('utf-8'))
```

Comment:
we have them almost everywhere... XD
It is for python3 compatibility. 
if your solution works with python3, when we start to clean the code, we could replace them in one time. (maybe using the [helpers](https://github.com/CERNDocumentServer/cds/blob/cdslabs_qa/tests/unit/helpers.py#L157) we already have)